web/wats/test/helpers/web.js

```diff
@@ -24,6 +24,7 @@ class Web {
     });
 
     this.page = await this.browser.newPage();
+    await this.page.setViewport({width: 1000, height: 1000})
```

Comment:
I'm OK with using this default and accepting that this means watsjs won't cover mobile resolutions by default, but puppeteer.launch takes a defaultViewport option ([docs](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#puppeteerlaunchoptions)), so it probably makes sense to set it above, rather than creating a page and resizing it every time you initialize a `Web`.